scripts/website/contentpreprocess.sh

```diff
@@ -89,8 +66,29 @@ EOT
 )" >${OUTPUT_CONTENT_DIR}/SECURITY.md
 cat SECURITY.md >>${OUTPUT_CONTENT_DIR}/SECURITY.md
 
-# Glob again to include new docs.
-ALL_DOC_CONTENT_FILES=$(echo "${OUTPUT_CONTENT_DIR}/**/*.md ${OUTPUT_CONTENT_DIR}/*.md")
+# Move root things to "thanos" and "contributing" dir for easy organizing.
+mkdir -p ${OUTPUT_CONTENT_DIR}/thanos/
+mv ${OUTPUT_CONTENT_DIR}/*.md ${OUTPUT_CONTENT_DIR}/thanos/
+mv ${OUTPUT_CONTENT_DIR}/thanos/CONTRIBUTING.md ${OUTPUT_CONTENT_DIR}/contributing/CONTRIBUTING.md
+mv ${OUTPUT_CONTENT_DIR}/thanos/CODE_OF_CONDUCT.md ${OUTPUT_CONTENT_DIR}/contributing/CODE_OF_CONDUCT.md
+mv ${OUTPUT_CONTENT_DIR}/thanos/community.md ${OUTPUT_CONTENT_DIR}/contributing/community.md
+
+# Add edit footer to all markdown files assumed as content.
+ALL_DOC_CONTENT_FILES=`echo "${OUTPUT_CONTENT_DIR}/**/*.md"`
+for file in ${ALL_DOC_CONTENT_FILES}
+do
+  relFile=${file##${OUTPUT_CONTENT_DIR}/}
+  echo "$(cat <<EOT
```

Comment:
The convention is to use `EOF`, for "End Of File"

Reply:
Alright. would effect changes as soon as possible